sdk/src/main/java/io/dapr/actors/runtime/ActorStatefulService.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import io.dapr.actors.ActorId;
+
+/**
+ * Implementation of the Actor Service that contains a state provider.
+ */
+class ActorStatefulService implements ActorService {

```

Comment:
This class should be visible to the user - it should be public.

Reply:
The idea is to minimize exposure to customers. It is possible to have the same flexibility via a configuration class.